test/powershell/Language/Scripting/ParameterBinding.Tests.ps1

```diff
@@ -1,7 +1,4 @@
-﻿Import-Module $PSScriptRoot\..\..\Common\Test.Helpers.psm1 -ErrorAction SilentlyContinue
```

Comment:
Adding a new helper modules and using of existing helper modules need to be documented. I believe this is enough.

Maybe @lzybkr  has something to add.